translations/default/AdminAdvparametersNotification.xlf

```diff
@@ -126,484 +126,271 @@
-        <target>Date (%date%) cannot be in the past (at line %line%). Format: %date_format%.</target>
-        <note>Line: 4070</note>
-      </trans-unit>
-      <trans-unit id="ef1a78ea7433cfc92f358d6dbf4fcd48">
-        <source>Format: Between 0 and 100</source>
-        <target>Format: Between 0 and 100</target>
-        <note>Line: 4236</note>
-      </trans-unit>
-      <trans-unit id="e0b985fb06e1498649ed1cfd6f3fb3e4">
-        <source>Discount rate (%rate%) is not valid (at line %line%). %format%.</source>
-        <target>Discount rate (%rate%) is not valid (at line %line%). %format%.</target>
-        <note>Line: 4074</note>
-      </trans-unit>
-      <trans-unit id="3f2f635ad300e97bc217b69cc06dfc30">
-        <source>Supply Order (%id%) is not editable (at line %line%).</source>
-        <target>Supply Order (%id%) is not editable (at line %line%).</target>
-        <note>Line: 4080</note>
-      </trans-unit>
-      <trans-unit id="ad1ce5290a697ed3729cbd520432cf22">
-        <source>Supply Order could not be saved (at line %line%).</source>
-        <target>Supply Order could not be saved (at line %line%).</target>
-        <note>Line: 4108</note>
-      </trans-unit>
-      <trans-unit id="bb19d4490bdc48656603c0a7ef30db73">
-        <source>Supply Order (%s) could not be loaded (at line %d).</source>
-        <target>Supply Order (%s) could not be loaded (at line %d).</target>
-        <note>Line: 4176</note>
-      </trans-unit>
-      <trans-unit id="5e063c49ce8203f2ca7a496da24b819e">
-        <source>Product/Attribute (%d/%d) cannot be added twice (at line %d).</source>
-        <target>Product/Attribute (%d/%d) cannot be added twice (at line %d).</target>
-        <note>Line: 4198</note>
-      </trans-unit>
-      <trans-unit id="0a657a7416c29bdc5b134b61b863e7ee">
-        <source>Product (%d/%d) is not available for this order (at line %d).</source>
-        <target>Product (%d/%d) is not available for this order (at line %d).</target>
-        <note>Line: 4211</note>
-      </trans-unit>
-      <trans-unit id="ca676c9ec4e1ae9fe9df3ac28910e6ef">
-        <source>Unit Price (tax excl.) (%d) is not valid (at line %d).</source>
-        <target>Unit Price (tax excl.) (%d) is not valid (at line %d).</target>
-        <note>Line: 4218</note>
-      </trans-unit>
-      <trans-unit id="406eba9770c82a84cae2d6ee3e846a56">
-        <source>Quantity Expected (%d) is not valid (at line %d).</source>
-        <target>Quantity Expected (%d) is not valid (at line %d).</target>
-        <note>Line: 4233</note>
-      </trans-unit>
-      <trans-unit id="8f7627c77b01dc968a9665c0156da405">
-        <source>Discount rate (%d) is not valid (at line %d). %s.</source>
-        <target>Discount rate (%d) is not valid (at line %d). %s.</target>
-        <note>Line: 4225</note>
+      <trans-unit id="32fa1d06b5d0a192c9991c4bf038b30c">
+        <source>Shop is not valid</source>
```

Comment:
What does it mean exactly?

Reply:
@Julievrz Warning when importing data and shop is not valid. I suggest installing something like VSCode, downloading a prestashop release and searching for a string in the code. You can find the context then.

![Snímek obrazovky 2024-05-23 101846](https://github.com/PrestaShop/PrestaShop/assets/6097524/c67ae699-2b5c-4631-af09-a2d63a50c72f)